packages/avatar/src/react/__stories__/index.story.tsx

```diff
@@ -1,7 +1,7 @@
 import { Meta, Story } from '@storybook/react/types-6-0'
 import React from 'react'
 
-import Avatar from '../index'
+import Avatar from '../../index'
```

Comment:
use src/index.ts (default/easy/styled file) instead of src/react/index.ts (unstyled)

Reply:
This pattern might present a problem when we write stories for sub-components that are not re-exported named or as statics. What do you think? Is this necessary for the next.js solution?

Reply:
This is not necessary for nextjs. This line is necessary for storybook to get the styled version of the component. 

If there are stories for sub-components, we'll want to import the `src/index.ts` file alongside. It's my prediction that will usually already be done.